src/App.js

```diff
@@ -14,7 +22,8 @@ class App extends React.Component {
             dataCard: {
                 statuses: null
             },
-            idCard: null
+            idCard: null,
+            redirect: null           
```

Comment:
I don't think you need redirect here, befoer answering please read about `Link` with `target="_blank"`